packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -37,6 +38,11 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
       throw new ConflictError(conflict);
     }
 
+    const currentVersion = data._version;
+    if (typeof (currentVersion) === "number") {
```

Comment:
Is it possible that the `_version` field will be of other type?

Reply:
Nope, this isn't actually needed, thanks for pointing out!

Reply:
Great!